CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# v31.0.0
+
+Kubernetes API Version: v1.31.1
+
+### API Change
+- The resource/v1alpha3.ResourceSliceList filed which should have been named "metadata" but was instead named "listMeta" is now properly "metadata". ([kubernetes/kubernetes#126761](https://github.com/kubernetes/kubernetes/pull/126761), [@thockin](https://github.com/thockin)) [SIG API Machinery]
```

Comment:
I didn't see any generated client change regarding this (https://github.com/kubernetes/kubernetes/pull/126761/files). Do you know why is that?

Reply:
we are release 1.31.0, https://github.com/kubernetes/kubernetes/releases/tag/v1.31.0

the release script is pulling the openapi from 1.31.1. i guess we can manually edit the CHANGELOG to reflect that we are actually release 1.31.0, WDYT?

Reply:
> i guess we can manually edit the CHANGELOG to reflect that we are actually release 1.31.0, WDYT?

Yes I think that works. Let's do that. Could you update this PR as well as https://github.com/kubernetes-client/python/pull/2282 and let me know when it's ready?

Reply:
done